ui/app/routes/vault/cluster/clients/history.js

```diff
@@ -26,24 +25,6 @@ export default class HistoryRoute extends Route {
     }
   }
 
-  async getVersionHistory() {
```

Comment:
this all comes now from the clients.js route, and is added to the routes model by modelFor

Reply:
At first I was wondering about this but now that I see the data is shared between both child routes so nice work loading it in the parent and accessing it this way!